Sources/ViewInspector/Inspector.swift

```diff
@@ -1,18 +1,115 @@
 import SwiftUI
 
 @available(iOS 13.0, macOS 10.15, tvOS 13.0, *)
-public struct Inspector { }
+public struct Inspector {
+
+    /// Removes the "(unknown context at <memory_address>)" portion of a type name.
+    /// Calls to this method are memoized and retained for the lifetime of the program.
+    /// - Parameter typeName: The raw type name. (e.g. ``SomeTypeName.(unknown context at $138b3290c).SomePropertyName``)
+    /// - Returns: The sanitized type name. (e.g. ``SomeTypeName.SomePropertyName``)
+    static func sanitizeNamespace(ofTypeName typeName: String) -> String {
+        var str = typeName
+
+        if let sanitized = sanitizedNamespacesCache[typeName] {
```

Comment:
I moved the exact implementation from the `String` extension to here and added the memoization calls here and in line 26.